src/plugins/workflow_designer/src/library/ConvertFilesFormatWorker.cpp

```diff
@@ -99,12 +99,11 @@ QVariantMap getFormatsMap(MapType mapType) {
         if (NULL == format || format->checkFlags(DocumentFormatFlag_CannotBeCreated)) {
             continue;
         }
-        if (format->checkFlags(DocumentFormatFlag_SupportWriting) || (BOOLEANS == mapType)) {
-            if (BOOLEANS == mapType) {
-                result[fid] = false;
-            } else {
-                result[fid] = fid;
-            }
+        const QString formatName = format->getFormatName();
+        if (BOOLEANS == mapType) {
+            result[formatName] = false;
```

Comment:
Worker should not deal with display values  (?)
The map returned here should use ids

Reply:
Why not? Only the worker knows what data should be in the combo box.

Reply:
Everywhere inside WD (runtime, uwl file) you should use 'format-id' and only in the visual components (combo box) show a 'rendered' form of the format-id. It is "no-change" from the current behavior.

The only change in this patch should be inside of the combo box component: it should take format id and show a display value for it from the map you added. Once user select a value in the combo box the result returned by the combo-box should have only format-ids as it is now.